custom/lambda/src/main/java/com/solarwinds/opentelemetry/extensions/PropertiesSupplier.java

```diff
@@ -30,6 +31,8 @@ public PropertiesSupplier() {
     defaultProperties.put(
         "otel.propagators", String.format("tracecontext,baggage,%s,xray", COMPONENT_NAME));
     defaultProperties.put("otel.instrumentation.runtime-telemetry.enabled", "false");
+    defaultProperties.put(
+        "otel.java.experimental.span-stacktrace.filter", SPAN_STACKTRACE_FILTER_CLASS);
```

Comment:
To confirm, if the end-user really wanted to they can supply their own filter class and override ours by setting the `otel.java.experimental.span-stacktrace.filter` system property, yes?

Reply:
yes